### ISSUE TITLE
Some features can't be used with just a keyboard

### ISSUE THREAD

**Issue by arielj** [Heroic-Games-Launcher/HeroicGamesLauncher#745]
**Describe the bug**

For accessibility reasons, it should be possible to use Heroic with just the keyboard, currently some things are just unreachable with a keyboard, and some are unclear when you are focusing them.

In the Library screen:
- as a Grid, each game card is an `a` tag, but when tabbing through them there's no indication on which is the current game card you are on. The currently focused card should trigger the game name overlay and the outline like it does in the list layout
- as a List, the play/download/settings icons are just SVGs, they should be inside `a` or `button` tags so they can be focused with the tab key
- in both layouts, the `Filter` selector has no outline to show you are focusing it with the keyboard
- in both layouts, the `refresh` icon is an SVG and can't be focused, it should probably be a `button` tag

In the GamePage screen:
- the Play Now / Install / Uninstall buttons have no outline so you can't tell if you are focusing them
- the `Install on ...` select has no outline
- in the submenu, only `Settings` can get focus, the rest of the elements (Verify/Move/Change/Log/Shortcuts/etc) are `span` tag and you can't focus them, they should be `a` or `button` tags
- the `Install Path:` and the `Prefix` path are clickable but one is a div and the other is a span, they should be `a` tags

In the Settings page:
- toggles and select tags have no online, you don't know what you are focusing
- the `Help !` element is clickable but can't be focused, it should probably be a button
- winecfg/winetricks/run exe on prefix elements are clickable but can't be focused
- the + / - / +folder can't be focused, they are only SVGs, they should probably be buttons

**To Reproduce**

Open Heroic and try to use only the keyboard to navigate the app using Tab, Shift+Tab, the arrow keys, and spacebar/enter to activate elements.

**Expected behavior**

Every clickable element should be also reachable with the keyboard.

**Desktop (please complete the following information):**

- OS: Linux Mint
- Heroic Version: latest main branch
- Game: -
- Wine/Proton version: -
- Hardware specs (RAM, CPU, GPU): -

**Comment by dragonDScript** [comment]
I think this is important to implement... If I have time I will do this

**Comment by arielj** [comment]
@dragonDScript are you working on this? I have some time over the next week and I want to fix this but not sure if you have made progress already

**Comment by flavioislima** [comment]
No, haven't looked at it.
Also, might be or not related but we also have an old PR to deal with joystick input. Feel free to pick that up if you want.
For now in focusing in

**Comment by flavioislima** [comment]
I think the keyboard and joystick features will work nice with the steam deck

**Comment by arielj** [comment]
I'll try the joystick input handling in a different PR, this one was alredy pretty big with all the accessibility changes

**Comment by flavioislima** [comment]
Yes, it's better 👍

**Comment by dragonDScript** [comment]
> @dragonDScript are you working on this? I have some time over the next week and I want to fix this but not sure if you have made progress already

nono i already have a lot of ongoing work...go with it
